pyclam/manifold.py

```diff
@@ -1267,7 +1301,6 @@ def _partition_threaded(self, criterion) -> List[Cluster]:
 
     def ancestry(self, cluster: Union[str, Cluster]) -> List[Cluster]:
         """ Returns the sequence of clusters that needs to be traversed to reach the requested cluster.
-
```

Comment:
Please undo the changes to the Doc-String formatting. We used the format recommended in PEP so that our doc-strings can be used to automatically generate documentation